### DISCUSSION TITLE
how i can use customable port ?

### DISCUSSION THREAD

**Discussion by abas** [spatie/db-dumper#148] [Q&A]
in some case i use different port with some custom port, how i can connect to db-server with custom port ?
an example
```bash
mysqldump -uuser -ppassword -hhost -P 3308 > out.sql 
```

**Comment by tahirkeskin** [comment]
Hi,

I wrote an example below. Use: "->setPort(3306)".

\Spatie\DbDumper\Databases\MySql::create()
                    ->setDbName('db_name')
                    ->setUserName('mysql_user')
                    ->setPassword('mysql_password')
                    ->setHost('server_ip')
                    **->setPort('3306')**
                    ->addExtraOption('--single-transaction')
                    ->useCompressor(new \Spatie\DbDumper\Compressors\GzipCompressor())
                    ->dumpToFile('path');

Have a nice day.
